docs/troubleshooting/kubernetes.rst

```diff
@@ -1,13 +1,223 @@
+
+BIG-IP Controller for Kubernetes/OpenShift
+------------------------------------------
+
+.. _common troubleshoot:
+
+Common Issues
+`````````````
+
+How do I check the settings of my |kctlr| using the command line?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use `kubectl`_ or `oc`_ commands to check the |kctlr| configurations using the command line.
+
+.. code-block:: console
+
+   kubectl [oc] get pod -o yaml [--namespace=kube-system]          \\ Returns the Pod's YAML settings
+   kubectl [oc] describe pod myBigIpCtlr [--namespace=kube-system] \\ Returns an information dump about the Pod you can use to troubleshoot specific issues
+
+-----------------------------------------
+
+What happened to my BIG-IP configuration changes?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you make changes to objects in the partition managed by the |kctlr| -- whether via configuration sync or manually -- **the Controller will overwrite them**. By design, the |kctlr| keeps the BIG-IP system in sync with what it knows to be the desired configuration. For this reason, F5 does not recommend making any manual changes to objects in the partition(s) managed by the |kctlr|.
+
+-----------------------------------------
+
+Why don't the |kctlr| pod(s) show up when I run ``kubectl get pods``?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you launched the |kctlr| in the ``--kube-system`` namespace, you should add the ``--namespace`` flag to your :command:`kubectl get` command.
+
+.. code-block:: console
+
+   kubectl [oc] get pods --namespace=kube-system
+   kubectl [oc] get pod myBigIpCtlr --namespace=kube-system
+
+-----------------------------------------
+
+Why do the BIG-IP pool members use the Kubernetes node IPs instead of the pod IPs?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The |kctlr| uses node IPs when running in its default mode, ``nodeport``. See :ref:`Nodeport mode vs Cluster mode <kctlr modes>` for more information.
+
+-----------------------------------------
+
+Why didn't the |kctlr| create any objects on the BIG-IP system?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Here are a few things to check:
+
+Does the namespace of the Kubernetes resource match the namespace(s) you set the Controller to watch?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: yaml
+   :caption: Excerpt from a sample Deployment
+
+   apiVersion: extensions/v1beta1
+   kind: Deployment
+   metadata:
+     name: k8s-bigip-ctlr-deployment
+     namespace: kube-system
+   ...
+             args: [
+               "--bigip-username=$(BIGIP_USERNAME)",
+               "--bigip-password=$(BIGIP_PASSWORD)",
+               "--bigip-url=10.190.24.171",
+               "--bigip-partition=kubernetes",
+               "--namespace=<my-namespace>",
```

Comment:
Would it help to highlight this particular line in the example so the reader knows what we are referring to?

Reply:
Added a comment as well